### PR TITLE
Update QuoteList.ts

### PR DESCRIPTION
**Pull request by fab33150**
# Pull Request

## Description:
Adds a new inspirational quote

## Related Issue:

Closes #XXXXX

### CODE REVIEW

**Review by gtchakama — DISMISSED** [review]

File: src/config/QuoteList.ts [hunk 1]
```diff
@@ -177,4 +177,8 @@ export const QuoteList: Quote[] = [
     text: "Develop success from failures. Discouragement and failure are two of the surest stepping stones to success.",
     author: "Dale Carnegie",
   },
+  {
+    text: "If you admire somebody you should go ahead and tell em, people never get their flowers while they can still smell em.",
+    author: "Kanye West",
+  }
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:183]:
```suggestion
  },
```
Needs a trailing comma. 😁

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by fab33150 — COMMENTED** [review]
added trailing comma

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**